views/task-items.html

```diff
@@ -0,0 +1,36 @@
+<div class="task-items-wrapper" ng-controller="itemsController">
+    <div ng-show='items.length == 0' class="no-tasks">There are no tasks available to you yet</div>
+
+    <div ng-repeat="item in items | filter: tasksFilter" class="task-item {{item.task.status}}" id="{{item.task.id}}"
+         ng-mouseenter="item.hovered = true;"
+         ng-mouseleave="item.hovered = false;">
+        <form name="taskItemForm" novalidate>
+            <div class="task-properties">
+                {{item.task.creationDate | datetime }}
+                <b>Assigned to:</b>
+                <input name="assignee" ng-show="item.access.edit" class='assign inline-edit' placeholder="Assign to"
+                       ng-model="item.task.assignee" required>
+                <input type="submit" class="btn assign-btn" ng-click="reassignTask(item)" value="Assign"
+                       ng-show="item.access.edit" ng-disabled="taskItemForm.assignee.$invalid">
+            </div>
```

Comment:
I pointed that css is preferable way. Here you need to duplicate item id ng-disabled="taskItemForm.assignee..."
ng-disabled="taskItemForm.description..."
this overloads code and makes it unreadable. 
with css you can handle all fields with single piece of code